src/lib/components/Login/Login.js

```diff
@@ -0,0 +1,124 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import Cookie from 'js-cookie';
+import { ONA, SupAuth } from 'gisida';
+import './Login.scss';
+import { Router } from '../../routes/router';
+
+const { defaultUnSupAuthZ: deAuthZ } = SupAuth;
+
+const mapStateToProps = (state) => {
+  const { APP } = state;
+  return {
+    appPassword: APP.password,
+    global: state,
+    appIcon: APP.appIcon,
+    loginIcon: APP.loginIcon,
+    publicUsername: APP.publicUsername,
+    publicPassword: APP.publicPassword
+  };
+};
+
+export const isLoggedIn = function() {
+  return Cookie.get('dsauth') === 'true';
+};
+
+export const logOut = (e) => {
+  e.preventDefault();
+  deAuthZ();
+  Cookie.set('dsauth', false);
+  Router.history.push('/login');
+};
+
+class Login extends Component {
```

Comment:
@ciremusyoka Let's name this component `OnaOauthLogin` because it is specifically for logging the user in via Ona oAuth.